pallets/common/src/lib.rs

```diff
@@ -90,14 +127,23 @@ pub mod erc;
 pub mod eth;
 pub mod weights;
 
+/// Weight info.
 pub type SelfWeightOf<T> = <T as Config>::WeightInfo;
 
+/// Collection handle contains information about collection data and id.
+/// Also provides functionality to count consumed gas.
+/// CollectionHandle is used as a generic wrapper for collections of all types.
```

Comment:
An additional line would be good for clarity and easier reading.

["...consumed gas.\n\nCollectionHandle is used..."]